pints/toy/_stochastic_degradation_model.py

```diff
@@ -0,0 +1,90 @@
+#
+# Logistic toy model.
+#
+# This file is part of PINTS.
+#  Copyright (c) 2017-2019, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+import numpy as np
+import scipy
+from scipy.interpolate import interp1d
+import pints
+
+from . import ToyModel
+
+
+class StochasticDegradationModel(pints.ForwardModel, ToyModel):
+
+    """
+    Stochastic decay model of a single chemical reaction [1].
+
+    Time until next reaction...
+    .. math::
+    $\tau = \frac{1}{A(t)k}*\ln{\frac{1}{r}}$
+
+    Has one parameter: Rate constant :math:`k`.
+    :math:`r` is a random variable, which is part of the stochastic model
+    The initial concentration :math:`A(0) = n_0` can be set using the
+    (optional) named constructor arg ``initial_concentration``
+
+    [1] Erban et al., 2007
+
+    *Extends:* :class:`pints.ForwardModel`, :class:`pints.toy.ToyModel`.
+    """
+
+    def __init__(self, initial_concentration=20):
+        super(StochasticDegradationModel, self).__init__()
+        self._n0 = float(initial_concentration)
+        if self._n0 < 0:
+            raise ValueError('Initial concentration cannot be negative.')
+
+    def n_parameters(self):
+        """ See :meth:`pints.ForwardModel.n_parameters()`. """
+        return 1
+
+    def simulate(self, parameters, times):
+        """ See :meth:`pints.ForwardModel.simulate()`. """
+        if parameters <= 0:
+            raise ValueError('rate constant must be positive')
```

Comment:
Make value error consistent with others in terms of capitalisation and full stops.